src/Campers.js

```diff
@@ -8,11 +9,28 @@ class Campers extends Component {
     var setup = this.props.camper.setup;
 
     this.state = { end: (date.substr(0,5) + ' ' + date.substr(9, date.length)), 
-                   techSetup: Array.isArray(setup) ? setup.join(', ') : setup};
+                   techSetup: Array.isArray(setup) ? setup.join(', ') : setup,
+                   id: this.props.camper_id,
+                   display: true };
+  
+    this.handleSubmit = this.handleSubmit.bind(this);
   }
   
+  handleSubmit(e) {
+    e.preventDefault();
+    const url = 'https://enigmatic-dawn-95873.herokuapp.com/api/v1/posts/' + this.props.camper._id;
```

Comment:
Look at the merged code in Campers.js - import server from config at the top and replace `http...` with `${server}` in URL.